src/main/java/com/example/service/UserServiceImpl.java

```diff
@@ -25,6 +31,10 @@ public UserResponseDto register(UserRegistrationRequestDto request)
         }
 
         User user = userMapper.toModel(request);
+        user.setPassword(passwordEncoder.encode(request.getPassword()));
+        Role userRole = roleRepository.findByRole(Role.RoleName.USER)
+                .orElseThrow(() -> new RuntimeException("Role USER not found"));
```

Comment:
use USER enum for exception message

```suggestion
                .orElseThrow(() -> new EntityNotFoundException("Role USER not found"));
```